### PR TITLE
fix syntax problem in CODEOWNERS file

### PR DESCRIPTION
**Pull request by FeynmanZhou**
The existing CODEOWNERS file is only valid for the GitHub ID defined in the last line. This PR is to fix the syntax problem in the CODEOWNERS file.

### CODE REVIEW

**Review by iamsamirzon — APPROVED** [review]

**Review by priteshbandi — DISMISSED** [review]
LGTM

File: CODEOWNERS [hunk 1]
```diff
@@ -1,13 +1,2 @@
-# Org-Level Owners (in alphabetical order)
-* @justincormack
```
Comment (yizha1) [inline CODEOWNERS:2]: Org level owners should be added to code owners as well.

Comment (FeynmanZhou) [reply CODEOWNERS:2]: Added

**Review by yizha1 — DISMISSED** [review]
LGTM

**Review by toddysm — APPROVED** [review]

**Review by priteshbandi — APPROVED** [review]

**Review by iamsamirzon — APPROVED** [review]

**Review by toddysm — APPROVED** [review]

**Review by vaninrao10 — APPROVED** [review]
LGTM

**Review by iamsamirzon — APPROVED** [review]

**Review by priteshbandi — APPROVED** [review]
LGTM

**Review by yizha1 — APPROVED** [review]
LGTM

**Pull request merged**